### PR TITLE
[APPS-3459] Update en.yml file with marketplace listing content

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n 

### Description

Update en.yml file with marketplace listing content.

### References
* JIRA: https://zendesk.atlassian.net/browse/APPS-3459

### Risks
* [low] - app content may not show on marketplace

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -6,6 +6,58 @@ packages:
...
+By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
+  - translation:
+      key: "txt.apps.iframe_app.app.installation_instructions"
+      title: "Installation Instructions for App: Iframe"
+      value: "To add the Iframe app to Zendesk Support:
+
+
+1. Sign in to your Zendesk Support account, then click the **Manage** icon at the bottom of the left nav bar.
+
+2. Select **Marketplace** from the Apps category.
```
Comment (svizzari) [inline translations/en.yml]: nit: This text varies a little between Built by Zendesk apps.  Sometimes it appears as

> Apps > Marketplace

Sometimes as

> Select **Marketplace** from the Apps category

Sometimes both Marketplace and Apps are bolded

**Comment by pmgrove** [comment]
@svizzari: removed unneeded installation instructions.

File: translations/en.yml [hunk 2]
```diff
@@ -6,6 +6,47 @@ packages:
...
+  - translation:
+      key: "txt.apps.iframe_app.app.long_description"
+      title: "Long Description for App: Iframe"
+      value: "An *iframe* is an HTML element that lets you nest one web page in another web page. The Zendesk Iframe app lets your agents quickly navigate to an external website and then return to their tickets without interrupting their workflow. Your agents can look up any external site and reference information such as order details or data on another system.
+
+
+The Iframe app presents the external source as an iframe in your Zendesk Support instance. Your agents can access this page by clicking the Iframe app icon.
+
+
+The Iframe app supports websites that include the [Zendesk App Framework (ZAF) SDK](https://github.com/zendesk/zendesk_app_framework_sdk). Check out [Iframes in Apps](https://developer.zendesk.com/apps/docs/agent/iframes_in_apps) to learn more.
```
Comment (HiroIguchi) [inline translations/en.yml]: Just to confirm. Apps Framework V1 will be deprecated: https://support.zendesk.com/hc/en-us/articles/115009518287
A link of "Iframes in Apps" is under Apps Framework v1 section in developer.zendesk.com. Will the link be changed later?

Comment (pmgrove) [inline translations/en.yml]: This is different to the ZAF v1 EOL so there is no need to update the content.

File: translations/en.json [hunk 3]
```diff
@@ -1,5 +1,21 @@
 {
   "app": {
+    "name": {
+      "title": "Name for App: Iframe",
+      "value": "Iframe"
```
Comment (mariasica) [inline translations/en.json]: Iframe or iFrame?

Comment (pmgrove) [inline translations/en.json]: It looks like we have always referred to it as the Iframe app: https://www.zendesk.com/apps/iframe/

**Comment by mariasica** [comment]
👍

**Pull request merged**